src/api/Client.ts

```diff
@@ -11,9 +11,8 @@ const getBaseURL = (network: Network) => {
     : process.env.WORMSCAN_TESTNET_API_BASE_URL;
 };
 
-const createNewClient = (baseURL: string) => {
-  client = createClient(baseURL);
-};
+const mainnetClient = createClient(getBaseURL("MAINNET"));
+const testnetClient = createClient(getBaseURL("TESTNET"));
```

Comment:
and here I think you could create multiple clients, and store them in a dict structure

```suggestion
const clients = {
   "MAINNET": createClient(getBaseURL("MAINNET")),
   "TESTNET": createClient(getBaseURL("TESTNET")),
}
```